docs/_articles/en/soql/soql-builder.md

```diff
@@ -111,21 +111,16 @@ Split your view to see your query in both SOQL Builder and the text editor.
 
 ![Split your screen to see both SOQL Builder and Text Editor](./images/soql-builder-split-panels.gif)
 
+## Save Query Results
+
+Click one of the Save icons to save the query results in a `.csv` or `.json` file in the location of your choice. To avoid deploying these files to your org or adding them in source control, remember to include any paths to saved files in the `.gitignore` file.
```

Comment:
Is this good now, @jonnyhork ?

Reply:
yeah that second part is great, thanks.

One thing that trips me up is this part 
> save the query results in a `.csv` or `.json` file in the location

The `in` and `in` is close for me, what about using `as` instead:
> save the query results as a `.csv` or `.json` file in the location

super nit pic but just thought I'd mention it.

Reply:
+1.

Reply:
+1 (majority rules!)

Reply:
Done, @jonnyhork. Ship it !?